internal/configuration/env_config.go

```diff
@@ -53,41 +54,42 @@ func init() {
 	settings["GetDebugMode"] = Setting{"JC_DEBUG_MODE", defaultDebugMode, []func(interface{}, string) error{util.IsBool}}
 }
 
+// Setting is an element in the proxy configuration. Many of such Setting constitutes full configuration.
```

Comment:
" Setting is an element in the proxy configuration. It contains the environment variable from which the setting is retrieved, its default value as well as a list of validations which the value of this setting needs to pass."